packages/roosterjs-editor-core/lib/coreApi/selectImage.ts

```diff
@@ -0,0 +1,36 @@
+import { createRange } from 'roosterjs-editor-dom';
+import { EditorCore, SelectImage, SelectionRangeTypes } from 'roosterjs-editor-types';
+
+/**
+ * @internal
+ * Select a image and save data of the selected range
+ * @param core The EditorCore object
+ * @param image Image to select
+ * @param wrapper Selected image wrapper
+ * @returns Selected image information
+ */
+export const selectImage: SelectImage = (
+    core: EditorCore,
+    image: HTMLImageElement | null,
+    wrapper?: HTMLSpanElement
+) => {
+    const selectedImage = image
```

Comment:
I think you need to ensure image has a unique id here

Reply:
Similar with selectTable.
Also ensure content DIV have id. Then we can use id and CSS to show image border.

Reply:
The id is incremental and reseted after dispose the editor? Why do we need use the id in the contentDiv?

Reply:
An example could be when re-adding the content with an image that already contains the ID.
Let's say you select an image, add the ID with number 0, if you copy and paste or set the content with that image to another editor where the incremental is still 0 there would be a duplicated ID